modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommercePaymentTerms.testcase

```diff
@@ -25,14 +25,51 @@ definition {
 		CommerceConfiguration.commerceInitializersTearDown();
 	}
 
-	@description = "[Please Update]"
+	@description = "Add a new payment term to Terms and Conditions."
 	@ignore = "Test Stub"
 	@priority = "4"
 	test AddANewPaymentTerm {
 		property portal.acceptance = "false";
 
 		// TODO COMMERCE-8279 AddANewPaymentTerm pending implementation
 
+		task ("Given a terms and coditions page.") {
+			CommerceNavigator.gotoPortlet(
+				category = "Order Management",
+				portlet = "Terms and Conditions");
+		}
+
+		task ("When a sales manager creates a new entry and is presented to the details page of the entry") {
+			CommercePaymentTerms.addTermsAndCondition(
+				termsAndConditionName = "Test Terms and Condition",
+				termsAndConditionType = "Payment Terms",
+				valuePosition = "");
+		}
+
+		task ("And when the sales manager goes back to the "Terms and Conditions" page, the new entry is present.") {
+			Click(locator1 = "Button#BACK");
+
+			CommercePaymentTerms.viewTermsAndConditionEntry(
+				termsAndConditionActiveStatus = "No",
+				termsAndConditionName = "Test Terms and Condition",
+				termsAndConditionType = "Payment Terms",
+				valuePosition = "0");
+		}
+
+		task ("Then the sales manager opens the new entry again and sees that all the field contains the correct data") {
```

Comment:
This should also be two tasks. "When the Sales Manager opens the new entry again" and

"Then the correct payment term data is shown""